src/ServiceControl.TransportAdapter/NServiceBusMetricReport.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Metrics;
+using Metrics.Json;
+using Metrics.MetricData;
+using Metrics.Reporters;
+using NServiceBus;
+using NServiceBus.Extensibility;
+using NServiceBus.Logging;
+using NServiceBus.Routing;
+using NServiceBus.Support;
+using NServiceBus.Transport;
+
+namespace ServiceControl.TransportAdapter
+{
+    using NServiceBus.Raw;
```

Comment:
Need to run the Particular code rules on this class (if not the whole codebase).